### PR TITLE
Highlight correct options for MRQ questions on submission page

### PR DESCRIPTION
**Pull request by jeremyyap**

### CODE REVIEW

File: client/app/bundles/course/assessment/submission/components/CheckboxFormGroup.jsx [hunk 1]
```diff
@@ -26,7 +27,10 @@ export default class CheckboxFormGroup extends Component {
               return input.onChange(newValue);
             }}
             label={(
-              <div dangerouslySetInnerHTML={{ __html: option.option.trim() }} />
+              <div
+                style={option.correct && readOnly ? { backgroundColor: green50 } : null}
```
Comment (allenwq) [inline client/app/bundles/course/assessment/submission/components/CheckboxFormGroup.jsx]: Should we only show this to staff?

Comment (jeremyyap) [reply client/app/bundles/course/assessment/submission/components/CheckboxFormGroup.jsx]: Backend will not send the flag if user is not staff https://github.com/Coursemology/coursemology2/blob/master/app/views/course/assessment/question/multiple_responses/_multiple_response.json.jbuilder#L6

Comment (allenwq) [reply client/app/bundles/course/assessment/submission/components/CheckboxFormGroup.jsx]: what if it's not correct?

Comment (jeremyyap) [reply client/app/bundles/course/assessment/submission/components/CheckboxFormGroup.jsx]: Then the option won't be highlighted? Correct here means option should be chosen, not student's answer is correct.

Comment (allenwq) [reply client/app/bundles/course/assessment/submission/components/CheckboxFormGroup.jsx]: OK, I recall.

**Pull request merged**